packages/traceability/issuer-api/migrations/1639660016195-AddSignerTable.ts

```diff
@@ -0,0 +1,49 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddSignerTable1639660016195 implements MigrationInterface {
+    name = 'AddSignerTable1639660016195';
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(
+            `CREATE TABLE "issuer_signer" ("blockchainNetId" integer NOT NULL, "platformOperatorPrivateKey" character varying NOT NULL, "isEncrypted" boolean NOT NULL, CONSTRAINT "PK_7c8b739645fb8518b866b45a8bb" PRIMARY KEY ("blockchainNetId"))`
```

Comment:
Why is the blockchainNetId needed for the signer?

Signer can sign messages/transactions without ever communicating with the blockchain.

Also, we don't really support multiple networks on the Issuer API level, so we will not have different signers for different networks.

If we force this table to only have a single row, and always use that signer, that should be enough for our use case.

Reply:
Ok, I will simplify this :-)

Reply:
My intention was different signer for different network, but yeah, since we don't support this, then I guess it's whatever.

Reply:
But... After all @JosephBagaric  I wonder if this is such a problem.
Because this is possible to create multiple blockchain properties (and it is, because this is not restricted anywhere, despite that it's impossible to use different blockchain properties in practice), why just don't keep this relation?
For me at least this would be more intuitive.

Reply:
@soanvig You're right, I thought it was restricted. Let's keep it then.